runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPlugin.java

```diff
@@ -1021,6 +962,14 @@ private void drawUi(final int canvasHeight, final int canvasWidth)
 		gl.glBindTexture(gl.GL_TEXTURE_2D, interfaceTexture);
 		gl.glUniform1i(uniTex, 0);
 
+		// If necessary, change the sampling function. Honestly, this should be using OpenGL
```

Comment:
I guess "change the sampling function" is what the code does, and what follows in the comment is potential improvement. Maybe those two parts can be split up with a newline to make it less confusing what the comment is trying to convey.

What the code does can probably be clarified, maybe "Sets the sampling function used when stretching the UI".

Reply:
Yeah comment there is a bit cryptic. Fixed that (and the line break). Also I was talking about [sampler objects](https://www.khronos.org/opengl/wiki/Sampler_Object). That's the _correct_ way to do it, but meh. Should I even leave that part of the comment in?